src/components/MatchProfilesForm/MatchProfilesForm.js

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
```

Comment:
Remove this file as this not needed in the scope of your story and not covered with tests.